lib/SPIRVProducerPass.cpp

```diff
@@ -4383,16 +4397,10 @@ void SPIRVProducerPassImpl::GenerateInstruction(Instruction &I) {
     // Find SPIRV instruction for parameter type.
     auto Ty = I.getType();
     if (Ty->isPointerTy()) {
-      auto PointeeTy = Ty->getPointerElementType();
-      if (PointeeTy->isStructTy() &&
```

Comment:
Interesting we can just delete this clause. But I think we we can.

Reply:
Right, this would be invalid SPIR-V anyways so why special case for it? If this ever came up as being generated, it should be fixed earlier in the flow.